supabase/lib/client_options.py

```diff
@@ -1,7 +1,9 @@
 from dataclasses import dataclass, field
-from typing import Any, Callable, Dict, Optional
+from typing import Any, Callable, Dict, Optional, Union
 
 from gotrue import SyncMemoryStorage, SyncSupportedStorage
+from postgrest.constants import DEFAULT_POSTGREST_CLIENT_TIMEOUT
+from httpx import Timeout
```

Comment:
Nitpicking but imports should be sorted alphabetically :p
```suggestion
from gotrue import SyncMemoryStorage, SyncSupportedStorage
from httpx import Timeout
from postgrest.constants import DEFAULT_POSTGREST_CLIENT_TIMEOUT
```
Apply this for the other file as well, thanks!

Reply:
Ohh my bad. Done ✅